app/javascript/mastodon/features/notifications/components/column_settings.js

```diff
@@ -66,4 +87,4 @@ class ColumnSettings extends React.PureComponent {
 
 }
 
-export default injectIntl(ColumnSettings);
+export default connect()(injectIntl(ColumnSettings));
```

Comment:
This component is already wrapped in ColumnSettingsContainer (that's where settings, onChange and onSave come from) so connecting it again directly here is undesired. Adjust the container component instead.